baybe/surrogates/gaussian_process.py

```diff
@@ -108,19 +108,17 @@ def _fit(self, searchspace: SearchSpace, train_x: Tensor, train_y: Tensor) -> No
 
         # If no kernel is provided, we construct one from our priors
         if self.kernel is None:
-            self.kernel = MaternKernel(lengthscale_prior=lengthscale_prior[0])
+            self.kernel = ScaleKernel(
+                base_kernel=MaternKernel(lengthscale_prior=lengthscale_prior[0]),
+                outputscale_prior=outputscale_prior[0],
+            )
 
         # define the covariance module for the numeric dimensions
-        gpytorch_kernel = self.kernel.to_gpytorch(
+        base_covar_module = self.kernel.to_gpytorch(
             ard_num_dims=train_x.shape[-1] - n_task_params,
             active_dims=numeric_idxs,
             batch_shape=batch_shape,
         )
-        base_covar_module = gpytorch.kernels.ScaleKernel(
-            gpytorch_kernel,
-            batch_shape=batch_shape,
-            outputscale_prior=outputscale_prior[0].to_gpytorch(),
-        )
         if outputscale_prior[1] is not None:
```

Comment:
can this (and `lengthscale_prior` below) become part of the baybe kernel classes? seems weird to select that here

Reply:
Just to be sure, you mean the "starting values" here, right?

Reply:
yes

Reply:
It was a bit of a pain since initial values cannot be handed over via `*args` and `**kwargs`, but I managed to get stuff included (although the code does not look nice, but I did not find an easier way). An alternative would be to overwrite the `to_gpytorch` functions of the individual classes, if you guys prefer that I could easily change this.

Reply:
Why so complicated? My (perhaps naive?) suggestion would be to keep only the code up to the `hasattr` check in the base class and then **partially** override `to_gpytorch` by calling the super method and just setting the initial value afterward. Or is there anything that speaks against it?

Reply:
Well, in our last discussions I had the feeling that you guys prefer to have the derived classes as easy and lean as possible such that it is *not* necessary to overwrite `to_gpytorch`, which is why I haven't implemented it that way. But if you prefer I can easily change that.

Reply:
I changed this now with the latest commit, please review and resolve if gucci.